data/data/vsphere/master/main.tf

```
@@ -22,6 +22,7 @@ resource "vsphere_virtual_machine" "vm_master" {
   folder               = var.folder
   enable_disk_uuid     = "true"
   annotation           = local.description
+  poweron_timeout      = var.vsphere_poweron_timeout
```

I think this entire PR can be:
```suggestion
  poweron_timeout      =  600
```
Again, sorry for the confusion about making this configurable.

I don't see much of a downside for adding this change. All it means is that when we do hit this error it is going to take 5 minutes longer for the install to fail.

@jcpowermac Do you think it's possible this could help with the BZ? i.e. are there cases where a VM doesn't power on for 5 minutes but might power on in, say, 7 minutes?

---

I was able to catch one of these virtual machines that experienced this issue and even hours later the allowed methods was missing poweron. So while I don't think 10 minutes will hurt anything I also don't think it will resolve anything either.

https://github.com/hashicorp/terraform-provider-vsphere/blob/fa3580820d1e27ff36419f87f8c2b487c6a167fa/vsphere/internal/helper/virtualmachine/virtual_machine_helper.go#L680

Occurrences the last 14 days in CI
https://search.ci.openshift.org/?search=PowerOnVM_Task&maxAge=336h&context=1&type=junit&name=&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job

I don't think we have any user reports of this issue though, right?